<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>990</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEdit_PidProcess">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>410</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>450</y>
      <width>251</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButton_continueProcess">
       <property name="text">
        <string>Continuar Processo</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton_StopProcess">
       <property name="text">
        <string>Parar Processo</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_KillProcess">
       <property name="text">
        <string>Finalizar Processo</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>461</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Procurar Processo  --&gt;</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="lineEdit_FindProcess"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="textBrowser_findProcess"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>10</y>
      <width>971</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Procesos Ativos:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_TimeUpdating">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Atualizar</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLineEdit" name="lineEdit_TimeUpdating"/>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTextBrowser" name="textBrowser_allProcess"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>400</y>
      <width>201</width>
      <height>161</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_5">
     <item row="1" column="0">
      <widget class="QTextBrowser" name="textBrowser_stopedProcess"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Historico</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelCPU">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>360</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>360</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>390</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Digite o PID</string>
    </property>
   </widget>
   <zorder>lineEdit_PidProcess</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>label_3</zorder>
   <zorder>labelCPU</zorder>
   <zorder>progressBar</zorder>
   <zorder>label_5</zorder>
   <zorder>label</zorder>
   <zorder>textBrowser_stopedProcess</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>990</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_KillProcess</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>killPid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_StopProcess</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>stopPid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>387</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_continueProcess</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>continuePid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>609</x>
     <y>455</y>
    </hint>
    <hint type="destinationlabel">
     <x>728</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_FindProcess</sender>
   <signal>textChanged(QString)</signal>
   <receiver>MainWindow</receiver>
   <slot>searchProcess()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_TimeUpdating</sender>
   <signal>clicked(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>timeUpdate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>67</y>
    </hint>
    <hint type="destinationlabel">
     <x>13</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>killPid()</slot>
  <slot>stopPid()</slot>
  <slot>continuePid()</slot>
  <slot>searchProcess()</slot>
  <slot>timeUpdate()</slot>
 </slots>
</ui>
